### DISCUSSION TITLE
Creating config file fails (mac)

### DISCUSSION THREAD

**Discussion by gudstrand** [sayanarijit/xplr#19] [Q&A] [answered]
╰─ xplr . | yq ".config" -y | tee ~/.config/xplr/config.yml
Error: unknown command ".config" for "yq"
Run 'yq --help' for usage.
thread 'main' panicked at 'failed printing to stdout: Broken pipe (os error 32)', library/std/src/io/stdio.rs:940:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

Tried version 3 of yq with same results.

**Comment by sayanarijit** [comment]
Can you export the config using the following?

```bash
xplr > app.yml && cat app.yml | yq ".config" -y | tee ~/.config/xplr/config.yml

# Press `#` when the app loads
```

**Reply by gudstrand** [reply]
Same error, although the app.yml seems to have content.  The yq command is failing.

**Reply by sayanarijit** [reply]
As per the [yq docs](https://github.com/mikefarah/yq#v4-released) it seems the v3 was quite different. Mine is version 2. Can you try version 2 or 4?

**Comment by sayanarijit** [comment, marked as answer]
With the latest version, you can specify only the `version: <version>` field, leaving the rest to default. Or you can [copy from here](https://github.com/sayanarijit/xplr/blob/main/src/config.yml).

I hope this resolves the issue.
